### PR TITLE
Updated readme to match reality.

### PR DESCRIPTION
**Pull request by rishson**
[ci skip]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,26 +56,29 @@ The CLI has the following format:
...
 `dojo -h, --help` - provides a list of help as detailed above.
 
 The CLI has the following in-built groups:
 
+`dojo create` - provides scaffolding for new Dojo 2 projects.
+`dojo eject` - allows users to configure and run command instead of the cli.
 `dojo version` - provides information on the versions of installed commands and the cli itself.
 
-`dojo create app` and `dojo build` are not installed by default with @dojo/cli. To use them, you must install them separately, e.g. with `npm i @dojo/cli-create-app -g`.
+`dojo build` and `dojo test` are not installed by default with `@dojo/cli`. To use them, you must install them separately, e.g. with `npm i @dojo/cli-build-webpack` and `npm i @dojo/cli-test-intern`. The reason for not bundling these 2 groups, is so you can run different versions of these group per-project.
```
Comment (dylans) [inline README.md]: "different versions of these group per-project" groups?

**Comment by rishson** [comment]
@dylans ready for another review when you have time 🎉

File: README.md [hunk 2]
```diff
@@ -6,13 +6,23 @@ The CLI is the officially supported way to create and maintain Dojo 2 apps.
...
+
+
+## Why use this cli?
+It is designed to save you time, by promoting a standardised workflow, and automating away more mundane boilerplate tasks.
+
+*Single dependency* - instead of having to download and configure multiple tools such as `Webpack`, `Intern` and `tslint`, you can just install the `cli` and know that all of these tools will work together.
+
+*Make the common tasks simple* - because you don't need to install and configure the individual tools yourself, you can be sure that the versions being used all work together and they they are running with sensible defaults.
+
+*Make the advanced tasks possible* - you can `eject` to a custom setup at any time. When you `eject`, all the configuration and build dependencies of the included tools will be moved into your project. If you are adept at configuring these tools, then you can now do so without the `cli` imposing its defaults.
```
Comment (dylans) [inline README.md]: imposing feels a bit more negative than intended... perhaps suggesting or using?

File: README.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ The CLI has the following in-built groups:
 `dojo eject` - allows users to configure and run command instead of the cli.
 `dojo version` - provides information on the versions of installed commands and the cli itself.
 
-`dojo build` and `dojo test` are not installed by default with `@dojo/cli`. To use them, you must install them separately, e.g. with `npm i @dojo/cli-build-webpack` and `npm i @dojo/cli-test-intern`. The reason for not bundling these 2 groups, is so you can run different versions of these group per-project.
+`dojo build` and `dojo test` are not installed by default with `@dojo/cli`. To use them, you must install them separately, e.g. with `npm i @dojo/cli-build-webpack` and `npm i @dojo/cli-test-intern`. The reason for not bundling these 2 groups, is so you can run different versions of these groups per-project.
```
Comment (dylans) [inline README.md]: But they are included when running dojo create with an app?

Comment (dylans) [inline README.md]: "The reason for not bundling these 2 groups, is so you can run different versions of these groups per-project." should perhaps say "These two groups are not included by default to allow different versions of these groups to be run per project."

Comment (rishson) [inline README.md]: They are not included and need installing per-project created with `create app`.

File: README.md (hunk 2 above)
Comment (rishson) [inline README.md]: will change to 'using'

**Comment by dylans** [comment]
@rishson care to also address #96 with this one?

**Pull request merged**